CMakeLists.txt

```diff
@@ -95,6 +95,15 @@ if(NOT K4GEO_USE_LCIO)
   message(STATUS "Use of LCIO is DISABLED, some detectors that depend on LCIO will not be built: ${lcio_sources}")
 endif()
 
+set(DCH_INFO_H "DCH_info.h" )
+set(FILES_DEPENDINGON_DCH_INFO_H "DriftChamber_o1_v02.cpp" )
+find_file(DCH_INFO_H_EXIST "${DCH_INFO_H}" PATHS "${DD4hep_INCLUDE_DIRS}/DDRec" NO_DEFAULT_PATH)
```

Comment:
```suggestion
include(CheckIncludeFileCXX)
set(CMAKE_REQUIRED_LIBRARIES DD4hep::DDRec)
CHECK_INCLUDE_FILE_CXX(DDRec/DCH_info.h DCH_INFO_H_EXIST)
set(CMAKE_REQUIRED_LIBRARIES)
set(FILES_DEPENDINGON_DCH_INFO_H "DriftChamber_o1_v02.cpp" )
```

Reply:
thank you very much @andresailer ! I tried out that macro but I was missing the previous step `set(CMAKE_REQUIRED_LIBRARIES DD4hep::DDRec)`!